plugins/proofpoint_tap/komand_proofpoint_tap/actions/parse_tap_alert/action.py

```diff
@@ -35,6 +36,8 @@ def run(self, params={}):
             )
         )
         if threat_details_urls:
-            clean_data["threat"]["threat_details_url"] = threat_details_urls[0]
-
-        return {Output.RESULTS: clean_data}
+            clean_data["threat"]["threatDetailsUrl"] = threat_details_urls[0]
+        clean_data["browser"] = clean(clean_data.get("browser", {}))
+        clean_data["message"] = clean(clean_data.get("message", {}))
+        clean_data["threat"] = clean(clean_data.get("threat", {}))
```

Comment:
The `clean` method used here uses the `return_non_empty` method. We did not want to call this method on the entire dict, because then the user would get an empty dict in the output, but now the user gets information about what types of information (even if no details were found) the alert may contain.